exercises/practice/resistor-color-duo/src/test/java/ResistorColorDuoTest.java

```diff
@@ -1,6 +1,7 @@
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.Ignore;
+import static org.assertj.core.api.Assertions.assertThat;
 
 import static org.junit.Assert.assertEquals;
```

Comment:
Error: eckstyle] [ERROR] /home/runner/work/java/java/exercises/practice/resistor-color-duo/src/test/java/ResistorColorDuoTest.java:6:15: Unused import - org.junit.Assert.assertEquals. [UnusedImports]

Reply:
Oops.  Not sure how I missed not committing that.  Should be fixed now.